app/models/notifications/mentioned_in_comment.rb

```diff
@@ -10,8 +10,15 @@ def deleted_translation_key
       "notifications.mentioned_in_comment_deleted"
     end
 
-    def self.filter_mentions(mentions, _mentionable, _recipient_user_ids)
-      mentions
+    def self.filter_mentions(mentions, mentionable, _recipient_user_ids)
+      ppl = mentionable.people_allowed_to_be_mentioned
+      if ppl == "all"
+        mentions
+      else
+        mentions.to_ary.delete_if {|mention|
```

Comment:
Right, but won't that be an inefficient `WHERE people.id IN (....)` expression? I remember before I was adviced to avoid this idiom in case when there are many ids in the list. Often there will be few mentions and a number of people ids (10, 100, even more).